config/glossary.json

```diff
@@ -79,6 +94,11 @@
     "definition": "A set of boundaries that encompass a directory, all of its users, and all of the applications which use the directory. In some implementations, this perimeter is a physical location; in others, it is a set of networks or devices connected via VPN.",
     "short": "A set of boundaries that encompass a directory, all of its users, and all of the applications which use the directory."
   },
+  "public-client": {
+    "title": "Public Client",
+    "definition": "According to the OAuth 2.0 protocol, clients (applications) can be classified as either confidential or public depending on whether or not they are able to hold credentials (such as a client ID and secret) securely. Public clients cannot hold credentials securely, so should only use grant types that do not require the use of their client secret. ID Tokens issued to them must be signed asymmetrically using a private key (RS256) and verified using the public key corresponding to the private key used to sign the token. See <a href=‘/applications/concepts/app-types-confidential-public'>Application Types - Confidential vs. Public</a>.",
```

Comment:
Add link to [RFC 6749 Section 2.1](https://tools.ietf.org/html/rfc6749#section-2.1)

Reply:
The glossary doesn't currently display links properly. When we get that bug fixed, we plan to go through and add links to specs and reference sections of our own docs.